cmake_format/configuration.py

```diff
@@ -229,6 +236,8 @@ def linewidth(self):
     "ruler_pattern":
     ("Regular expression to match rulers in comments default=r'{}'"
      .format(DEFAULT_RULER_PATTERN)),
+    "ignore_directories":
+    "list of directories which will be ignore during parsing, use ':' as separator directory",
```

Comment:
@dkandyba will be ignored

Reply:
@AByzhynar Fixed in: 87b6fc709025c5bd8eb8be595494c09cbc6207cc